### DISCUSSION TITLE
How can I complete path as zsh?

### DISCUSSION THREAD

**Discussion by otakutyrant** [nushell/nushell#12360] [Q&A]
As you see, it is painful that I have to hit "Enter" to complete everytime in Nushell, meanwhile it is unnecessary in Zsh.

https://github.com/nushell/nushell/assets/64188229/81e45fac-534b-4e2f-b125-1731a1fa9a0e

**Comment by fdncred** [comment]
I think the closest we get to this behavior right now is with `$env.config.completions.quick` set to `true`. If there is only 1 option and you hit tab to complete, it autofills it.

I believe the completions and hit menu behavior are part of reedline.

**Reply by otakutyrant** [reply]
OMG, I hope there are more close behavior we can get.

**Reply by fdncred** [reply]
All it takes is a contributor to create a good PR.

**Reply by otakutyrant** [reply]
You are right. I hope there is one day that I can start to contribute.

**Comment by lafleurdeboum** [comment]
Hey, I'd like to second that - I made #18919 to request that same feature (if I'm correct). But now I suspect it's doable using the keybindings setup.
